dnd5esheets/front/src/5esheets-client/core/OpenAPI.ts

```diff
@@ -20,7 +20,7 @@ export type OpenAPIConfig = {
 };
 
 export const OpenAPI: OpenAPIConfig = {
-    BASE: process.env.NODE_ENV === 'production' ? '/' : 'http://127.0.0.1:8000',
+    BASE: process.env.NODE_ENV === 'production' ? '' : 'http://127.0.0.1:8000',
```

Comment:
Due to the fact that you removed the Makefile rule for this file, next time it's generated, it'll be reverted back to `BASE: ''` which is alright, now that we have an SPA, I think.

```suggestion
    BASE: '',
```

Reply:
Ah! I missed that. Let's merge it anyway, and we'll find a fix later.

Reply:
I think that you can just accept the suggestion and we should be alright

Reply:
fixed by: https://github.com/brouberol/5esheets/pull/158/commits/4d69abe279fb2f5dfbcbd38c0a0738a3657d691c

> I think that you can just accept the suggestion and we should be alright

We still need the conditional somewhere.